{
  "Title": "16s Metagenomic analysis of Gut Microbiome",
  "link_to_issue": "https://github.com/brainhackorg/global2022/issues/101",
  "issue_number": 101,
  "labels": [
    {
      "name": "marseille_fra",
      "description": "Marseille event",
      "color": "d4c5f9"
    },
    {
      "name": "programming:documentation",
      "description": "Markdown, Sphinx",
      "color": "5319e7"
    },
    {
      "name": "project",
      "description": "",
      "color": "f9bc70"
    },
    {
      "name": "programming:Python",
      "description": "",
      "color": "5319e7"
    },
    {
      "name": "programming:Unix_command_line",
      "description": "",
      "color": "5319e7"
    },
    {
      "name": "project_type:documentation",
      "description": "",
      "color": "c5def5"
    },
    {
      "name": "programming:shell_scripting",
      "description": "",
      "color": "5319e7"
    },
    {
      "name": "status:published",
      "description": "",
      "color": "0e8a16"
    },
    {
      "name": "programming:R",
      "description": "",
      "color": "5319e7"
    },
    {
      "name": "project_development_status:0_concept_no_content",
      "description": "",
      "color": "bfd4f2"
    },
    {
      "name": "project_tools_skills:familiar",
      "description": "",
      "color": "c2e0c6"
    },
    {
      "name": "status:web_ready",
      "description": "",
      "color": "0e8a16"
    },
    {
      "name": "topic:reproducible_scientific_methods",
      "description": "",
      "color": "006b75"
    }
  ],
  "content": "### Title\r\n\r\n16s Metagenomic analysis of Gut Microbiome \r\n\r\n### Leaders\r\n\r\nDipankar Bachar\r\n\r\n### Collaborators\r\n\r\n_No response_\r\n\r\n### Brainhack Global 2022 Event\r\n\r\nBrainhack Marseille\r\n\r\n### Project Description\r\n\r\n16S rRNA gene sequencing is commonly used for identification, classification and quantitation of microbes within complex biological mixtures such as environmental samples and gut samples. Many studies [1][2] are trying to establish the link between the intestinal microbiota with different neurodevelopment, neurogical disorders etc. In these studies the BioInformatic analysis of the data coming from gut samples plays a very important role. So, in our brainhack project we will work on the publicly available data from one of these article[1]. So, this project is aimed for those who are interested in the BioInformatic analysis of Gut Microbiota and are also interested in the different pipelines and analysis tools like QIIME2[4] , cutadapt[5] , Kraken2[6] etc.\r\n\r\nIn this project, we propose:\r\n    To go through this [1] article to understand their method of analyzing the gut microbiota\r\n    To download their 16s sequencing data from NCBI ( submitted by the authors ) \r\n    To extract and filter the data \r\n\r\nFor advanced users, We also propose:\r\n    To write( Python ) few tools for filtering the sequencing data\r\n    To try to create an Anaconda environment or a Singularity image with the 16s analysis pipeline QIIME2[4]\r\n    To try to implement some of the methods of analysis with QIIME2 ( as described in the article )\r\n    To understand the importance of reproducibility of the results\r\n\r\nReferences :\r\n\r\n[1] The microbial metabolite p-Cresol induces autistic-like behaviors in mice by remodeling the gut microbiota\r\nPatricia Bermudez-Martin et al.\r\n\r\n[2]  Gut Microbiota Regulate Motor Deficits and Neuroinflammation in a Model of Parkinson's Disease\r\nTimothy R Sampson  et al.\r\n\r\n[3] https://lcsciences.com/16s-rrna-gene-sequencing-for-identification-classification-and-quantitation-of-microbes/\r\n\r\n[4] Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2\r\nEvan Bolye et al.\r\n    \r\n[5] https://cutadapt.readthedocs.io/en/stable/\r\n    \r\n[6] https://ccb.jhu.edu/software/kraken2/\r\n\r\n### Link to project repository/sources\r\n\r\nhttps://github.com/dumaatravaie/BrainHack_Marseille_2022_16s\r\n\r\n### Goals for Brainhack Global\r\n\r\nThe goal of this project to have a basic idea about how the BioInformatics analysis of \r\n16S rRNA sequences are done and also to familiarize with different freely available tools and pipelines \r\nused for the analysis. \r\n\r\n\r\n### Good first issues\r\n\r\n- Installation of anaconda and create a conda environment\r\n- Creation of Image Singularity\r\n\r\n### Communication channels\r\n\r\nhttps://mattermost.brainhack.org/brainhack/channels/brainhack_marseille_2022_16s_metagenomic\r\n\r\n### Skills\r\n\r\nAnaconda or Singularity 50%\r\nPython 50%\r\nUnix/Linux 25%\r\nBasic knowledge about 16s sequencing 25%\r\n\r\n### Onboarding documentation\r\n\r\n_No response_\r\n\r\n### What will participants learn?\r\n\r\nThey will have the basic idea about how the BioInformatic analysis of 16s metagenomic data are done. They will have a preliminary idea about how to search, download and use the open source sequence data from NCBI. They will also learn about many of the state of art open source software tools and pipelines ( which are used for the analysis of 16s data ).\r\n\r\n### Data to use\r\n\r\n_No response_\r\n\r\n### Number of collaborators\r\n\r\n2\r\n\r\n### Credit to collaborators\r\n\r\nAll of the collaborators or participants will have equal credit ...\r\n\r\n### Image\r\n\r\n![image](https://user-images.githubusercontent.com/9570817/199976131-f9ccaa3d-ee11-4ebc-8c62-86281e589e21.png)\r\n\r\n\r\n### Type\r\n\r\nother\r\n\r\n### Development status\r\n\r\n0_concept_no_content\r\n\r\n### Topic\r\n\r\nreproducible_scientific_methods, other\r\n\r\n### Tools\r\n\r\nother\r\n\r\n### Programming language\r\n\r\nPython, `R`, shell_scripting, unix_command_line\r\n\r\n### Modalities\r\n\r\nnot_applicable\r\n\r\n### Git skills\r\n\r\n4_not_applicable\r\n\r\n### Anything else?\r\n\r\nThis is a BioInformatics project where we will focus on a published article and will work on the data submitted by the authors to a public data repository ( NCBI ) ...\r\n\r\n### Things to do after the project is submitted and ready to review.\r\n\r\n- [X] Add a comment below the main post of your issue saying: `Hi @brainhackorg/project-monitors my project is ready!`\r\n- [ ] Twitter-sized summary of your project pitch.",
  "project_url": "https://github.com/dumaatravaie/BrainHack_Marseille_2022_16s",
  "project_description": "\r\n\r\n16S rRNA gene sequencing is commonly used for identification, classification and quantitation of microbes within complex biological mixtures such as environmental samples and gut samples. Many studies [1][2] are trying to establish the link between the intestinal microbiota with different neurodevelopment, neurogical disorders etc. In these studies the BioInformatic analysis of the data coming from gut samples plays a very important role. So, in our brainhack project we will work on the publicly available data from one of these article[1]. So, this project is aimed for those who are interested in the BioInformatic analysis of Gut Microbiota and are also interested in the different pipelines and analysis tools like QIIME2[4] , cutadapt[5] , Kraken2[6] etc.\r\n\r\nIn this project, we propose:\r\n    To go through this [1] article to understand their method of analyzing the gut microbiota\r\n    To download their 16s sequencing data from NCBI ( submitted by the authors ) \r\n    To extract and filter the data \r\n\r\nFor advanced users, We also propose:\r\n    To write( Python ) few tools for filtering the sequencing data\r\n    To try to create an Anaconda environment or a Singularity image with the 16s analysis pipeline QIIME2[4]\r\n    To try to implement some of the methods of analysis with QIIME2 ( as described in the article )\r\n    To understand the importance of reproducibility of the results\r\n\r\nReferences :\r\n\r\n[1] The microbial metabolite p-Cresol induces autistic-like behaviors in mice by remodeling the gut microbiota\r\nPatricia Bermudez-Martin et al.\r\n\r\n[2]  Gut Microbiota Regulate Motor Deficits and Neuroinflammation in a Model of Parkinson's Disease\r\nTimothy R Sampson  et al.\r\n\r\n[3] https://lcsciences.com/16s-rrna-gene-sequencing-for-identification-classification-and-quantitation-of-microbes/\r\n\r\n[4] Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2\r\nEvan Bolye et al.\r\n    \r\n[5] https://cutadapt.readthedocs.io/en/stable/\r\n    \r\n[6] https://ccb.jhu.edu/software/kraken2/\r\n\r\n"
}
